docs/docsite/rst/reference_appendices/general_precedence.rst

```diff
@@ -133,4 +121,11 @@ These variables don't survive the playbook object they were defined in and will
 
 And there is also a 'host scope' - variables that are directly associated with the host (also available via the `hostvars[]` dictionary). The host  scope variables are  available across plays and are  defined in inventory, vars plugins, or from modules (set_fact, include_vars).
 
-All of the above makes knowing where to define a variable and its overrides a complex subject, but only if you start using many different ways to define a variable. Normally you only use a few and just need to know the interactions between those methods.
+Using ``-e`` extra variables at the command line
+------------------------------------------------
+
+To override all other settings in all other categories, you can use extra variables: ``--extra-vars`` or ``-e`` at the command line. Values passed with ``-e`` are variables, not command-line options, and they will override command-line options as well as variables set elsewhere. For example this task will connect as ``brian`` not as ``carol``::
```

Comment:
isn't this covered already in variable precedence doc?

Reply:
It is covered there. I thought it was worth covering again here for two reasons. First, extra vars are a bit confusing (it's a variable, but because you pass it on the command line, it "feels" like a command-line option). Second, because users may find this page when they're looking for a quick fix for a setting that isn't getting set the way they want - and giving them the "big hammer" option right there on the same page may help them.

Reply:
+1 for keeping this here. As a precedence newbie, it was an important point for me to learn.